CMakeLists.txt

```diff
@@ -51,6 +51,34 @@ endif()
 list(APPEND CMAKE_MODULE_PATH ${LLVM_CMAKE_DIR})
 include(AddLLVM)
 
+# ROCM version from hipify sources
+# Maintain a record of the HIP latest version in CMakeLists.txt as well
+set(HIP_VERSION_MAJOR_DEFAULT 6)
+set(HIP_VERSION_MINOR_DEFAULT 4)
+set(HIP_VERSION_PATCH_DEFAULT 0)
+
+# Check if rocm is installed and grab rocm version if installed, else use the default values set above
+set(ROCM_VERSION_FILE /opt/rocm/share/hip/version)
```

Comment:
We need to support all the supported OSs.

Reply:
Got it, need a solution for windows as well.

Reply:
> Got it, need a solution for windows as well.

Have you been able to come to a solution? Do you need assistance?

Reply:
Hi, I'm thinking about it.. Will it only apply to cases where HIP_SDK is installed? Otherwise we can just consider a standalone HIPIFY installation and provide the version from HIPIFY sources?